homeassistant/components/environment_canada/weather.py

```diff
@@ -184,23 +184,34 @@ def get_forecast(ec_data, forecast_type):
 
     if forecast_type == "daily":
         half_days = ec_data.daily_forecasts
+
+        today = {
+            ATTR_FORECAST_TIME: dt.now().isoformat(),
```

Comment:
The time should be UTC time.

https://developers.home-assistant.io/docs/core/entity/weather#forecast